### DISCUSSION TITLE
Is it possible to use multiple remotes with one hub?

### DISCUSSION THREAD

**Discussion by ReubenH12** [pybricks/support#1424] [Q&A]
I bought a few train remotes becuase I figured I could connect them all to one hub, and then sometimes I could have that hub broadcast to others if necessary. However I've been trying to connect them all and it's not working. Of course I could just connect one remote to each hub and broadcast between the two, but for my current project and likely future ones I would like more remotes than I have hubs. I figured I should ask if it's possible first before I spend ages on it. I've looked around for answers but nothing seems to say exactly whether you can.

**Comment by laurensvalk** [comment]
> but for my current project and likely future ones I would like more remotes than I have hubs

Would you like more remotes, or just more buttons/functions?

While each hub can connect to only one remote, you can add some logic to "switch modes" using the center button.

For example, drive using the +/- buttons, then click the center button to operate a crane using the +/- buttons instead. Click it again to drive, etc.

This should not take more time than moving your hands to another remote :)

-------------------

> I've looked around for answers but nothing seems to say exactly whether you can.

Thanks, that's useful feedback. The block language does indicate this now, but the `BUSY` error in Python probably isn't very descriptive. (It means Bluetooth is already busy with another remote.)

![image](https://github.com/pybricks/support/assets/12326241/5b2797fb-2b65-4a08-a2cf-e0b3ed6496a4)

**Reply by ReubenH12** [reply]
Ok thank you, that's useful. I guess I'll just have a mode switching button. 

I understand it's probably not a priority, but in the future would it ever be possible to connect multiple remotes, or is it a limitaion of the hardware and Lego?

**Reply by laurensvalk** [reply]
The hubs have various different Bluetooth chips; some are better at "multitasking" than others.

Prime / Inventor / Essential could probably handle N remotes. Maybe the Technic / City Hub could do 2 remotes in theory.

So far though, we've found that most applications do quite well with just one remote per hub.

We might also have support for other remote types in the future with more buttons, in which case a single remote might suffice.

**Reply by ReubenH12** [reply]
Oh ok, thanks for the info.
